apps/docs/ui.inc

```diff
@@ -0,0 +1,27 @@
+;;;;;;;;;;;;;;;;;;;;;;;;
+; user interface actions
+;;;;;;;;;;;;;;;;;;;;;;;;
+
+(import "././edit/ui.inc")
+
+;overload Edit app ui.inc
+(defun action-file-leaf-action ()
+	;load up the file selected
+	(defq file (. *file_tree* :get_route
+		(. *window* :find_id (getf *msg* +ev_msg_action_source_id))))
+	(select-node file)
+	(populate-page file))
+
+(defun action-close ()
+	(setq *running* :nil))
+
+(defun action-minimise ()
+	(bind '(x y w h) (apply view-fit
+		(cat (. *window* :get_pos) (. *window* :pref_size))))
+	(. *window* :change_dirty x y w h))
+
+(defun action-maximise ()
+	(bind '(x y) (. *window* :get_pos))
+	(bind '(w h) (. *window* :pref_size))
+	(bind '(x y w h) (view-fit x y (* w 2) (* h 2)))
```

Comment:
I suspect that makeing the width `(* w 2)` won't have any effect toher than leave a nasty band down the right side of the page ?

Reply:
strangely that doesn't happen but the minimize and maximize seem to be inverted. Investigating.

Reply:
ok fixed min / max